Cargo.toml

```diff
@@ -28,8 +28,8 @@ uuid = { version = "1.4.1", features = [
 ] }
 num-traits = "0.2.17"
 dotenv = "0.15.0"
-tycho-core = { git = "https://github.com/propeller-heads/tycho-indexer.git", package = "tycho-core", tag = "0.41.0" }
-tycho-client = { git = "https://github.com/propeller-heads/tycho-indexer.git", package = "tycho-client", tag = "0.41.0" }
+tycho-core = { git = "https://github.com/propeller-heads/tycho-indexer.git", package = "tycho-core", rev = "baf2619d8021b6ebcfe06a04774249c79aba25e9"  }
+tycho-client = { git = "https://github.com/propeller-heads/tycho-indexer.git", package = "tycho-client", rev = "baf2619d8021b6ebcfe06a04774249c79aba25e9"  }
```

Comment:
This can be changed back once we merge the pr in the tycho indexer repo

Reply:
I think it would be better to do things the other way around, ntq should use tycho-simulation version. We could think about re-exporting tycho-core on the tycho-simulation crate to avoid accidentally using two different versions in dependencies: https://doc.rust-lang.org/rustdoc/write-documentation/re-exports.html